src/pages/hyeriShin/Main/MainShin.scss

```
@@ -0,0 +1,405 @@
+            }
+          }
+        }
+
+        .feeds_comment_like_btn_box {
+          display: flex;
+          justify-content: space-between;
+
+          .like_btn {
+            width: 3%;
+            margin-left: auto;
+          }
+        }
+
+        .comment_box {
+          display: flex;
+          justify-content: space-between;
+          margin: 1rem 0;
+        }
+
+        .comment_input {
+          width: 100%;
+
+          form {
+            display: flex;
+            width: 100%;
+
+            input,
+            button {
+              width: 100%;
+              padding: 0.8rem 0;
+              border: 1px solid #efefef;
+              border-left: transparent;
+              border-right: transparent;
+              border-bottom: transparent;
+              background: white;
+            }
+
+            input {
+              width: 90%;
+              padding-left: 1rem;
+              color: #8f8f8f;
+            }
+
+            button {
+              width: 10%;
+              color: #b8dffc;
+            }
+          }
+        }
+      }
+    }
+    .sub_container {
+      position: sticky;
+      top: 6rem;
+      max-width: 293px;
+      margin-left: 2rem;
+
+      .feeds_profile {
+        display: flex;
+        margin-bottom: 1.3rem;
+
+        .feed_comment_small_img {
+          max-width: 20%;
+          border-radius: 50%;
+          border: 1px solid #e6e6e6;
+        }
+
+        .small_info {
+          display: flex;
+          justify-content: center;
+          flex-direction: column;
+          margin-left: 1rem;
+
+          .feeds_user_info {
+            margin-bottom: 0.3rem;
+            font-weight: bold;
+          }
+        }
+      }
+
+      .feeds_story_container,
+      .feeds_recommend_container {
+        width: 100%;
+        padding: 1rem 0;
+        border: 1px solid #e6e6e6;
+        background: white;
+      }
+
+      .feeds_story_container {
+        margin-bottom: 1rem;
+      }
+
+      .feeds_story_box,
+      .feeds_recommend_box {
+        width: 90%;
+        margin: 0 auto;
+      }
+
+      .feeds_box_txt {
+        display: flex;
+        justify-content: space-between;
+        margin-bottom: 1rem;
+        font-size: 0.8rem;
+        color: #a7a7a7;
+        font-weight: 600;
+
+        p {
+          a {
+            color: black;
+          }
+        }
+      }
+
+      .story_users {
+        .story_users_info {
+          display: flex;
+          margin: 0.6rem 0;
+
+          img {
+            max-width: 15%;
+            border-radius: 50%;
+          }
+        }
+
+        .story_users_info {
+          .story_users_info_txt {
+            margin-left: 0.5rem;
+            color: #9e9e9e;
+            p {
+              span {
+                margin-top: 0.2rem;
+                font-size: 0.9rem;
+              }
+            }
+          }
+        }
+      }
+
+      .feeds_recommend_box {
+        .recommend_users_info {
```

```suggestion
        .recommend_users_info {
```
스타일 없는 선택자를 굳이 써줄 필요가 있을까요?

---

앗.. 지운다는게 잊고있었네요  수정했습니다 감사합니다 ! 🥲